### DISCUSSION TITLE
Ended because API returned an empty page.

### DISCUSSION THREAD

**Discussion by twardy95** [victornpb/undiscord#449] [1. Questions & Answers] [answered]
Again the same error that's been bothering a lot of people ever since (#312, #290, #284, #277). Everything was working fine for me until yesterday when I reached messages from around 6 months ago. I've tried many methods mentioned in older posts but none of them worked for me. The script just does not work anymore and says "Ended because API returned an empty page." no matter what.

Any ways to fix this?
![image](https://user-images.githubusercontent.com/47645650/222221700-079af78c-8dce-4694-9dfe-81eeaee22206.png)

**Comment by Kilvoctu** [comment, marked as answer]
https://discordstatus.com/incidents/3hnnd9j68hgv

Discord search API went down yesterday. Supposedly resolved, but I wouldn't be surprised if there are lingering issues.

Yeah seem like something is still borked at Discord. Deleting messages manually, and the official client is not updating correctly. Pages and pages of nothing.  
![image](https://user-images.githubusercontent.com/2993060/222280548-f51ecfef-493d-47df-be49-e89455691969.png)

**Reply by twardy95** [reply]
Damn alright then I hope it will get fixed soon

**Reply by Kilvoctu** [reply]
Me too, I got 30k messages to delete, lol.

**Reply by yumam** [reply]
Lol I spent an hour trying to debug the code. Should've just came here first smh

**Reply by yeahrighthaha** [reply]
rip i thought i was having chrome issues

**Reply by yumam** [reply]
It's working again!

**Reply by twardy95** [reply]
yep it's working again

**Reply by tommytheturtle1** [reply]
> Me too, I got 30k messages to delete, lol.
How are you doing it? I am getting API limit error and my delays is 2100 and 2100 
i tried doing 100 and 1000(delete), any suggestions how i can speed this process?
